spec/clojure_ttt/board_spec.clj

```diff
@@ -29,8 +29,11 @@
   (it "should know if there is a draw"
```

Comment:
You use the words `should know` a lot, I would either simplify that to `knows` or `determines`
e.g. `(it 'determines if a position is valid')`